### ISSUE TITLE
pruner warning

### ISSUE THREAD

**Issue by koen84** [aptos-labs/aptos-ait2#24]
My logs are repeating a bunch of these.

`2022-07-16T00:41:51.830458Z [aptosdb_pruner] WARN storage/aptosdb/src/pruner/state_store/mod.rs:141 State pruner inconsistent, min_readable_version is 1109283 and  index_min_non-purged_version is 1109284`

**Comment by sherry-x** [comment]
Yes we're aware of this and the team is working on a fix for this warning, thanks for letting us know! cc @msmouse

**Comment by msmouse** [comment]
The latest hotfix (aptoslabs/validator:testnet_f5d8013b0a1851da8e078394d83130d3adaf7670) fixes this
